### PR TITLE
Removes TODOs and unused code

### PR DESCRIPTION
**Pull request by amprokop**
Removes the PumaThreadLogger, which I'm 99% sure we no longer want, and a bunch of TODOs and outdated comments.

### CODE REVIEW

File: app/mappers/address_mapper.rb [hunk 1]
```diff
@@ -6,7 +6,6 @@ def get_address_from_bgs_address(bgs_address = {})
       address_line_3: bgs_address[:addrs_three_txt],
       city: bgs_address[:city_nm],
       country: bgs_address[:cntry_nm],
-      # is this an odd name for state?
```
Comment (sharonwarner) [inline app/mappers/address_mapper.rb]: Yeah... why is that the name for state??

File: client/app/certification/constants/constants.js [hunk 2]
```diff
@@ -24,13 +24,6 @@ export const SHOW_VALIDATION_ERRORS = 'SHOW_VALIDATION_ERRORS';
 export const RESET_STATE = 'RESET_STATE';
 
 // types of hearings
-//
-// TODO:
-// on the backend, NO_HEARING_DESIRED
-// should result in VIDEO being written to VACOLS,
-// and HEARING_CANCELLED should result in a cancellation
-// checkbox being checked, but the original hearing type
-// should be undisturbed.
```
Comment (sharonwarner) [inline client/app/certification/constants/constants.js]: Do we do this?

File: app/mappers/address_mapper.rb (hunk 1 above)
Comment (amprokop) [inline app/mappers/address_mapper.rb]: Probably because there are non-state values that can be in here.

Comment (amprokop) [inline app/mappers/address_mapper.rb]: Like military bases maybe

**Pull request closed without merging**